test/it/positive.spec.js

```diff
@@ -0,0 +1,15 @@
+const compiler = require("../utils/compiler");
+const listSubDirectories = require("../utils/listSubDirectories");
+const POSITIVE_ROOT_PATH = "test/it/code-samples/positive";
+const positiveRoots = listSubDirectories(POSITIVE_ROOT_PATH);
+
+describe("typescript - positive scenarios", () => {
+  it.each(positiveRoots)(
+    "should successfully compile %s folder",
+    async tsRootPath => {
+      const configurationFilename = `${POSITIVE_ROOT_PATH}/${tsRootPath}/tsconfig.json`;
```

Comment:
if "listSubDirectories" would just concat the subdir to the root dir, you will not need to "join" here